core/includes/theme.inc

```diff
@@ -1896,7 +1896,10 @@ function theme_links($variables) {
 
       $class = array();
       // Use the array key as class name.
-      $class[] = backdrop_html_class($key);
+      $key_class = backdrop_html_class($key);
+      if ($key_class != '_') {
```

Comment:
Isn't that very specific to the styleguide module?

That module's quite buggy, although usable (I use it regularly).

Reply:
Not really. Any call to `theme_links()` that doesn't include array keys will cause this:

```php
dpm(theme_links(array('links' => array(
  array(
    'title' => 'one',
    'href' => '<front>',
  ),
  array(
    'title' => 'two',
    'href' => '<front>',
  ),
  array(
    'title' => 'three',
    'href' => '<front>',
  ),
))));
```

Reply:
> Any call to theme_links() that doesn't include array keys...

That's a weird example, completely against the [official docs](https://docs.backdropcms.org/api/backdrop/core%21includes%21theme.inc/function/theme_links/1).

Your example throws two php Notices. Is the styleguide module doing it that way?

Reply:
I don't know why the above throws those notices, I thought the missing things were optional...